### PR TITLE
new package: py-usgs

### PR DESCRIPTION
**Pull request by Sinan81**
new package py-usgs

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-usgs/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class PyUsgs(PythonPackage):
+    """Client library for interfacing with USGS datasets"""
+
+    homepage = "https://github.com/kapadia/usgs"
+    url      = "https://pypi.org/packages/source/u/usgs/usgs-0.2.7.tar.gz"
+
+    version('0.2.7', sha256='484e569ea1baf9574e11ccf15219957364690dcf06ee3d09afef030df944e79b')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-mock', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-usgs/package.py]:
```suggestion
    depends_on('py-mock', type='test')
```

File: var/spack/repos/builtin/packages/py-usgs/package.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    depends_on('py-pytest@2.8.7', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-usgs/package.py]:
```suggestion
    depends_on('py-pytest@2.8.7', type='test')
```
Where did you find the pytest version dependency info?

**Comment by Sinan81** [comment]
pytest dependency is mentioned in requirements fie: https://github.com/kapadia/usgs/blob/master/requirements.txt

**Comment by Sinan81** [comment]
@adamjstewart a random question: I've noticed recently you have been working on QT packages, and also you're doing remote sensing work. I am wondering if you have any plans on creating QGIS package, or else I think I will be  working on that soon. Thanks

**Comment by adamjstewart** [comment]
@Sinan81 I wasn't planning on writing a `qgis` package but I would love to see one! Let me know when you finish it and I can test it.

**Pull request merged**